test/python/ExampleCircuit.qasm

```diff
@@ -0,0 +1,8 @@
+OPENQASM 2.0;
+include "qelib1.inc";
+qreg q[1];
+creg c[1];
+
+x q[0];
+
+measure q[0] -> c[0];
```

Comment:
Please just use Qiskit in the tests to create this simple circuit and dump it to QASM if you really need a QASM file.

Reply:
Done